js/ui/scheduler/workspaces/view_data_provider.js

```diff
@@ -231,24 +232,24 @@ class ViewDataGenerator {
         return groupedDataMap;
     }
 
-    _isFirstGroupCell(rowIndex, columnIndex, singleGroupRowCount, singleGroupColumnCount, groupCount) {
+    _isFirstGroupCell(rowIndex, columnIndex, singleGroupRowCount, singleGroupColumnCount, groupCount, groupOrientation) {
```

Comment:
I suggest using `options` to reduce the number of arguments: 
`_isFirstGroupCell(rowIndex, columnIndex, options)`